recipes/_compile.rb

```diff
@@ -78,6 +78,9 @@
   #   https://github.com/CentOS/sig-cloud-instance-images/issues/4
   #
   package 'tar'
+  # Don't introduce a bug on RHEL by not having bzip2
+  # this was fixed previously in omnibus v2.4.12
+  package 'bzip2'
```

Comment:
Sorry, I seem to be writing a lot of comments about comments lately, but:

* I'm missing context on why this comment is needed.  Is removal of the `bzip2` something that keeps biting us?
* Assuming that it is needed, I'd put that context in the comment.